app/views/dashboard/landing_page.html.slim

```diff
@@ -1,7 +1,7 @@
 -unless params[:search]
   .carousel
     -@collections.each do |c|
-      -cache c do
+      -cache [c, :old_landing_page] do
```

Comment:
Here, passing the actual object `c` rather than `c.id`. This is to account for, when there are changes in `c`. 

If we just do `c.id` in key, then if there are changes, say new pages/works added, it will not be invalidated.

Passing `c` object, I believe, is being etag cached by rails by it's `updated_at`. So any changes in the collection SHOULD invalidate the cache here.